### DISCUSSION TITLE
Along with nuts - often pecans - and sugar, what is the other essential ingredient in traditional praline?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#202] [Galaxy Trivia] [answered]
**Hint:** 🧈

**Comment by muink** [comment]
Common is cream

**Reply by ParanoidUser** [reply]
You're absolutely right, cream can be used. But in the traditional recipe, there is a "heavier" type of cream. Also, cream itself has this ingredient as an addition to milk. Can you guess? 😊

**Comment by Faiqlatheef** [comment]
The other essential ingredient in traditional praline, along with nuts (often pecans) and sugar, is cream. Cream is used to give the praline its smooth and creamy texture. The nuts are usually roasted and then mixed with sugar and cream, and then boiled until the mixture thickens and turns golden brown. The mixture is then poured onto a sheet pan and allowed to cool and harden into a candy-like confection. Pralines are a popular sweet treat in the Southern United States, particularly in New Orleans, Louisiana, where they are a staple of the local cuisine.

**Comment by DurandSacha** [comment, marked as answer]
The other essential ingredient in traditional praline is usually almonds. Praline is a culinary preparation made from dried fruit (most often hazelnuts or almonds)

**Reply by ParanoidUser** [reply]
Nice try, @DurandSacha! Any ideas what that hint emoji (🧈) shows? 😊

**Reply by DurandSacha** [reply]
Ah yes !! this is a butter !

**Reply by ParanoidUser** [reply]
> Ah yes !! this is a butter !  

Well done! One more answer to go for the Galaxy Brain badge. Good luck! 😊

**Reply by DurandSacha** [reply]
thanks !
